src/tests/ftest/util/command_utils_base.py

```diff
@@ -524,6 +524,10 @@ def _get_new(self):
         """
         return CommandWithParameters(self.namespace, self._command, self._path)
 
+    def command_prepend_il(self, libpath):
+        """Add LD_PRELOAD=libpath into command to enable interception library"""
+        self._command = f'LD_PRELOAD={libpath} {self._command}'
+
```

Comment:
#11921 Just landed to master, so we can remove this now

Reply:
Thank you! Fixed as you suggested.